docs/lago-tutorial/README.rst

```diff
@@ -0,0 +1,242 @@
+   By default Lago will use the file from: http://templates.ovirt.org/repo/repo.metadata\
+   
+-  init.json, the name of the file which describes our environment.
+
+Auto configurations
+
+::
+
+    lago init
+
+This command will use the following default configurations:
+
+-  The command will search for a file named "LagoInitFile" that will be
+   used as the init file.
+-  The workdir will be named ".lago"
+-  As mentioned above, The default repo will be used.
+
+Note: Lago will copy the deployment scripts into the new environment, and set up the following environment variables:
+
+- $LAGO_PREFIX_PATH = FULL_PATH/.lago/current
+- $LAGO_WORKDIR_PATH = FULL_PATH/.lago
+
+so when using a relative path to the deployment (in the init file), there is no worry that 
+the path will be broken when trying to deploy the vms from a directory that doesn't satisfy the relative path.
+In general, all the other commands of lago can use those variables.
+
+Deploy the VMs
+^^^^^^^^^^^^^^
+
+When using the manual configurations all the commands should be invoked
+from /lago-work-dir.
+
+When using the default workdir you can invoke commands from it's parent,
+or any 'sibiling', for example:
+
+::
+
+    lago init
+    mkdir lolo
+    cd lolo
+    lago status
+
+Now, lets start the vms:
+
+::
+
+    lago start
+
+Or for a specific vm named "server":
+
+::
+
+    lago start server
+
+The command below will run the deployment scripts (from within the
+vms) that were specified in the init.json file.
+
+so, if you used the "manual configurations", the command should be
+invoked from /lago-work-dir.
+
+::
+
+    lago deploy
+
+
+-  Jenkins will be installed on the server.
+-  OpenJDK will be installed on the slaves.
+
+Getting the state of the environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You cae get information about the state of the enviorment with:
+
+::
+
+    lago status
+
+Or for a formated output as json or yaml:
+
+::
+
+    lago --out-format [json | yaml] status
+
+You can write down to yourself the ip adresses of the server and
+slaves, because we will need them when configuring the server.
+
+Interacting with the VMs
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Lago allows you to connect to the vms via ssh.
+for exmaple, if we have a vm named "server" we will use the following:
+
+::
+
+    lago shell server
+
+If the deployment scripts run successfuly we don't have
+to connect to the machines.
+
+In case of a failure, you can access the vms via console.
+This is useful when the vm failed to boot or when trubleshooting
+network issues.
+
+::
+
+    lago console server
+
+
+Stop the environment
+^^^^^^^^^^^^^^^^^^^^
+
+In order to stop the machines (brute shutdown) we will use:
+
+::
+
+    lago stop
+
+Or for a specific vm named "server":
+
+::
+
+    lago stop server
+
+Removing the enviornment
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following command will remove all the files
+that relates to the environment.
+
+::
+
+    lago destroy
+
+
+More Configurations 
```

Comment:
great, once you add the top-level title to the scenarios/jenkins/README it will show great here (you might want to use maxdepth 2, to allow showing also sublevels, like 'about' and such